packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -122,7 +122,7 @@ export default class QuanticSearchBox extends LightningElement {
           queries: getItemFromLocalStorage(this.localStorageKey) ?? [],
         },
         options: {
-          maxLength: 100,
+          maxLength: 20,
```

Comment:
is 20 too small? I agree 100 is too much, but 20 seems borderline, what do you think? I was thinking 50 unless you think the performance issues were impacted by the max length vs # of calls made for `getQuerySuggestionsNotInRecentQueries `

Reply:
I agree, 20 is more conservative